### PR TITLE
Cleanup format

### PR DESCRIPTION
**Pull request by KhafraDev**
FormData is better now, made Response/Request start with `Response {` or `Request {`, etc.

```js
> var fd = new FormData()
> fd.set('a', 'b')
> fd.append('a', new Blob(['hello']), 'hello.txt')
> fd.set('file', new Blob(['file']))
> fd

FormData {
  a: [
    'b',
    File { size: 5, type: '', name: 'hello.txt', lastModified: 1710432489080 }
  ],
  file: File { size: 4, type: '', name: 'blob', lastModified: 1710432492961 }
}
```

### CODE REVIEW

File: lib/web/fetch/formdata.js [hunk 1]
```diff
@@ -157,12 +157,24 @@ class FormData {
...
+          a[b.name].push(b.value)
+        } else {
+          a[b.name] = [a[b.name], b.value]
+        }
+      } else {
+        a[b.name] = b.value
+      }
+
+      return a
+    }, {})
```
Comment (Uzlopak) [inline lib/web/fetch/formdata.js:172]: What about prototype pollution?

Comment (KhafraDev) [reply lib/web/fetch/formdata.js:172]: doesn't matter, there are ten thousand areas vulnerable to prototype pollution

Comment (Uzlopak) [reply lib/web/fetch/formdata.js:172]: You joking, right?

Comment (KhafraDev) [reply lib/web/fetch/formdata.js:172]: no, we don't use primordials anywhere in the codebase (nor should we). it's a non-issue.

Comment (KhafraDev) [reply lib/web/fetch/formdata.js:172]: oh I see what you're talking about... I thought you meant more generally

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Uzlopak — APPROVED** [review]
Lgtm

**Pull request merged**